posthog/api/decide.py

```diff
@@ -149,7 +149,7 @@ def get_decide(request: HttpRequest):
 
             feature_flags, _, feature_flag_payloads, errors = get_all_feature_flags(
                 team.pk,
-                data["distinct_id"],
+                str(data["distinct_id"]),
```

Comment:
huh, this seems a bit adhoc. Is distinct_id not a consistent type?

Reply:
So, distinctID is always a string, but when you `JSON.parse("12345")`, it returns the number 12345 instead of the string, so need to coerce back.

This isn't usually a problem as implicit casting was happening elsewhere from time to time, but in an array of distinctIDs, this broke down as one had string while the other int type

Yep, very annoying, this is the second sentry error I linked to above.

So now, instead of implicit type casting, I made it explicit.

Reply:
Oh, and based on the payload, looks like sometimes an int is sent from the source, when they mean a string, and we coerce this somewhere in the ingestion pipeline 😬